src/demo-app/system-config.ts

```diff
@@ -50,6 +50,7 @@ System.config({
     '@angular/cdk/bidi': 'dist/packages/cdk/bidi/index.js',
     '@angular/cdk/coercion': 'dist/packages/cdk/coercion/index.js',
     '@angular/cdk/collections': 'dist/packages/cdk/collections/index.js',
+    '@angular/cdk/datepicker': 'dist/packages/cdk/datepicker/index.js',
```

Comment:
While we're at it, the same should be done for the e2e app. That way it'll be easier when we start writing e2e tests for it.